util/separators.lua

```diff
@@ -8,10 +8,11 @@
 
 local wibox = require("wibox")
 local gears = require("gears")
+local beautiful = require("beautiful")
 
 -- Lain Cairo separators util submodule
 -- lain.util.separators
-local separators = { height = 0, width = 9 }
+local separators = { height = (beautiful.separators_height or 0), width = (beautiful.separators_width or 9) }
```

Comment:
Round parentheses are not necessary, please remove them.

Reply:
Fixed I hope